docs/getting-started/zowe_v2_faq.md

```diff
@@ -6,4 +6,30 @@ The official date is TBD, the target is April 25, 2022; look for the official an
 
 ## Where can I find the current (V1) and new (V2) LTS conformancee criteria? 
 
-The Zowe Squads have prepared XLS spreadsheets with conformance criteria for all Zowe extensions including: CLI, APIs, App Framework, and Explorerfor VS Code. The spreadsheets clearly show the prior / V1 criteria alongside the new / V2 criteria. Please be aware, there are additions, deletions, and CHANGES to the criteria. In some cases the change is simply that a BEST PRACTICE has been deemed REQUIRED. Use the light-GREEN highlights to easily identify the changes. See the Changes to the [Conformance Criteria](https://www.zowe.org/vnext#conformance-changes) section at Zowe.org/vNext.
\ No newline at end of file
+The Zowe Squads have prepared XLS spreadsheets with conformance criteria for all Zowe extensions including: CLI, APIs, App Framework, and Explorerfor VS Code. The spreadsheets clearly show the prior / V1 criteria alongside the new / V2 criteria. Please be aware, there are additions, deletions, and CHANGES to the criteria. In some cases the change is simply that a BEST PRACTICE has been deemed REQUIRED. Use the light-GREEN highlights to easily identify the changes. See the Changes to the [Conformance Criteria](https://www.zowe.org/vnext#conformance-changes) section at Zowe.org/vNext.
+
+## Whats the difference between "server.json" and "example-zowe.yaml"?
+
+The previous Zowe V1.x config, "server.json", has been removed from V2 and has been replaced with a new yaml configuration file. The app server will no longer support instances/workspaces which only contain a "server.json" config file and will fallback to a default configuration. In addition to the app server, ZSS will no longer support "server.json".
+
+The yaml Zowe configuration file contains configurations for the setup, install, and initialization of Zowe as well as for individual components. This file allows users to customize dataset names, security related configs, certificate setup/config, job name & job prefix, various runtime configs, high availability config, as well as individual component configurations.
+
+For more information on Zowe setup and the yaml configuration, run the following command in the command line:
+
+  zwe init --help
+
+## What are the new default ports?
+
+Two of the default Zowe ports have changed: the app server and zss. The new default app server port is 7556 (previously 8544) and the new zss port is 7557 (previously 8542).
```

Comment:
There's many other ports that changed. You should change this to say "What are the new App framework default ports?" OR you can change this to say "Many default ports have changed in V2. You can compare the changes by viewing the old set here (https://github.com/zowe/zowe-install-packaging/blob/v1.x/master/scripts/instance.template.env) and the new set here (https://github.com/zowe/zowe-install-packaging/blob/v2.x/staging/example-zowe.yaml) or something like this

Reply:
I added all the changed portsnnow, cross referenced both of those files